lib/taste_tester/host.rb

```diff
@@ -26,6 +26,29 @@
 require 'taste_tester/exceptions'
 
 module TasteTester
+  # Wrapper around IO to stream, prefixing the lines with the hostname
+  class IO < IO
```

Comment:
Makes this a little easier to understand
```suggestion
  class IO < ::IO
```